### PR TITLE
[CLD-3946] Add support for EBS CSI addon in EKS

### PR DESCRIPTION
**Pull request by stylianosrigas**
#### Summary
- Add support for EBS CSI addon in EKS

#### Release Note

```release-note
- Add support for EBS CSI addon in EKS
```

### CODE REVIEW

**Review by spirosoik — DISMISSED** [review]

**Comment by spirosoik** [comment]
@stylianosrigas lint is failing

**Comment by stylianosrigas** [comment]
> @stylianosrigas lint is failing

Yeah trying to find out why. Its not code related.

**Comment by spirosoik** [comment]
> > @stylianosrigas lint is failing
> 
> Yeah trying to find out why. Its not code related.

seems related to version of aws provider and tf lint? not clear

**Review by angeloskyratzakos — APPROVED** [review]

File: aws/atlantis/providers.tf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 terraform {
+  required_version = ">= 1.1.8"
```
Comment (stafot) [inline aws/atlantis/providers.tf:2]: @stylianosrigas is it possible to make this variable? If not I am fine as it is.

Comment (stylianosrigas) [reply aws/atlantis/providers.tf:2]: @stafot unfortunately you cannot put a variable in this section

**Review by angeloskyratzakos — APPROVED** [review]

**Review by stafot — APPROVED** [review]
lgtm

**Pull request merged**